aio/content/examples/getting-started/src/app/product-alerts/product-alerts.component.1.ts

```diff
@@ -1,11 +1,13 @@
 // #docplaster
-// #docregion as-generated, imports
+/*
+// #docregion as-generated
 import { Component, OnInit } from '@angular/core';
 // #enddocregion as-generated
-import { Input } from '@angular/core';
+*/
+// #docregion as-generated, imports
+import { Component, OnInit, Input } from '@angular/core';
 import { Product } from '../products';
 // #enddocregion imports
```

Comment:
Oops my mistake. This should be:

```suggestion
// #docregion imports
import { Component, OnInit, Input } from '@angular/core';
import { Product } from '../products';
// #enddocregion imports
// #docregion as-generated
```

Reply:
updated. Thanks.